ethereumj-core/src/main/java/org/ethereum/samples/BasicSample.java

```diff
@@ -85,18 +91,27 @@ public BasicSample(String loggerName) {
     }
 
     protected void setupLogging() {
-        // Turn off all logging to stdout except of sample logging
-        LogManager.getRootLogger().removeAppender("stdout");
-        ConsoleAppender appender = new ConsoleAppender(stdoutAppender.getLayout());
-        appender.setName("stdout");
-        appender.setThreshold(Level.ERROR);
-        LogManager.getRootLogger().addAppender(appender);
-        logger = LoggerFactory.getLogger(loggerName);
-        LogManager.getLogger(loggerName).addAppender(stdoutAppender);
+        logger = createLoggerFor(loggerName);
     }
 
-    private void removeErrorLogging() {
-        LogManager.getRootLogger().removeAppender("stdout");
+    private static Logger createLoggerFor(String loggerName) {
+        LoggerContext lc = (LoggerContext) LoggerFactory.getILoggerFactory();
+        PatternLayoutEncoder ple = new PatternLayoutEncoder();
+
+        ple.setPattern("%d{HH:mm:ss.SSS} [%c{1}]  %m%n");
+        ple.setContext(lc);
+        ple.start();
+        ConsoleAppender<ILoggingEvent> consoleAppender = new ConsoleAppender<>();
+        consoleAppender.setEncoder(ple);
+        consoleAppender.setContext(lc);
+        consoleAppender.start();
+
+        ch.qos.logback.classic.Logger logger = (ch.qos.logback.classic.Logger) LoggerFactory.getLogger(loggerName);
+        logger.addAppender(consoleAppender);
+        logger.setLevel(Level.DEBUG);
+        logger.setAdditive(false); /* set to true if root should log too */
+
+        return logger;
     }
```

Comment:
The idea behind tweaking logger settings was to start the logger as usual then switch console appender to output only sample logger messages and set ERROR level for others
When running `BasicSample` I still see all other loggers INFO messages